dronekit/cameras.py

```diff
@@ -0,0 +1,138 @@
+        logging.info(log_msg)
+        pass
+
+
+def send_http_cmd(cmd):
+    try:
+        with Timeout(1):
+            data = urlopen(cmd).read()
+            return data
+    except Exception as e:
+            errprinter('>>> Exception in http command: ' + str(e))
+            return str(e)
+
+
+class GoProHero3(object):
+    def __init__(self, wifi_password="goprohero3", geotag=True, myvehicle=None):
+        self.wifipassword = wifi_password
+        self.num_picture = 0
+        self.geotagger = None
+        if geotag:
+            self.geotagger = GeoTagger(vehicle=myvehicle)
+
+        # See https://github.com/KonradIT/goprowifihack/blob/master/WiFi-Commands.mkdn
+        # for a list of http commands to control the GoPro camera
+
+        #TODO: convert these into methods.
+        self.on = "http://10.5.5.9/bacpac/PW?t=" + self.wifipassword + "&p=%01"
+        self.off = "http://10.5.5.9/bacpac/PW?t=" + self.wifipassword + "&p=%00"
+        self.stop = "http://10.5.5.9/bacpac/SH?t=" + self.wifipassword + "&p=%00"
+        self.videoMode = "http://10.5.5.9/camera/CM?t=" + self.wifipassword + "&p=%00"
+        self.burstMode = "http://10.5.5.9/camera/CM?t=" + self.wifipassword + "&p=%02"
+        self.timeLapseMode = "http://10.5.5.9/camera/CM?t=" + self.wifipassword + "&p=%03"
+        self.previewOn = "http://10.5.5.9/camera/PV?t=" + self.wifipassword + "&p=%02"
+        self.previewOff = "http://10.5.5.9/camera/PV?t=" + self.wifipassword + "&p=%00"
+        self.wvga60 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%00"
+        self.wvga120 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%01"
+        self.v720p30 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%02"
+        self.v720p60 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%03"
+        self.v960p30 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%04"
+        self.v960p48 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%05"
+        self.v1080p30 = "http://10.5.5.9/camera/VR?t=" + self.wifipassword + "&p=%06"
+        self.viewWide = "http://10.5.5.9/camera/FV?t=" + self.wifipassword + "&p=%00"
+        self.viewMedium = "http://10.5.5.9/camera/FV?t=" + self.wifipassword + "&p=%01"
+        self.viewNarrow = "http://10.5.5.9/camera/FV?t=" + self.wifipassword + "&p=%02"
+        self.res11mpWide = "http://10.5.5.9/camera/PR?t=" + self.wifipassword + "&p=%00"
+        self.res8mpMedium = "http://10.5.5.9/camera/PR?t=" + self.wifipassword + "&p=%01"
+        self.res5mpWide = "http://10.5.5.9/camera/PR?t=" + self.wifipassword + "&p=%02"
+        self.res5mpMedium = "http://10.5.5.9/camera/PR?t=" + self.wifipassword + "&p=%03"
+        self.noSound = "http://10.5.5.9/camera/BS?t=" + self.wifipassword + "&p=%00"
+        self.sound70 = "http://10.5.5.9/camera/BS?t=" + self.wifipassword + "&p=%01"
+        self.sound100 = "http://10.5.5.9/camera/BS?t=" + self.wifipassword + "&p=%02"
```

Comment:
are you planning on adding all of these to the gopro interface? I wonder if we should have a generic `command` class which is handling all of them, and expose `shutter` as a top level user of `command('shutter')`?